arrow-docs/docs/arrow/typeclasses/traverse/README.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: docs-core
+title: Traverse
+permalink: /arrow/typeclasses/traverse/
+---
+
+## Traverse
+
+
```

Comment:
Additional empty lines

Reply:
I checked and there is exactly 4 lines in `Functor`, `Foldable`, and `Monad`, etc. So I concluded it's okey to have them in this doc too. lmkwdyt.

Reply:
weird. Okay, let's roll with it.